iac-extensions/spring-config/src/test/java/org/sonar/iac/springconfig/checks/ExcessiveFileUploadSizeLimitCheckTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.springconfig.checks;
+
+import javax.annotation.Nullable;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.sonar.iac.springconfig.utils.SpringConfigVerifier;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ExcessiveFileUploadSizeLimitCheckTest {
+  @Test
+  void shouldDetectSensitiveValueInProperties() {
+    SpringConfigVerifier.verify("ExcessiveFileUploadSizeLimitCheck/ExcessiveFileUploadSizeLimitCheck.properties", new ExcessiveFileUploadSizeLimitCheck());
+  }
+
+  @Test
+  void shouldDetectSensitiveValueInYaml() {
+    SpringConfigVerifier.verify("ExcessiveFileUploadSizeLimitCheck/ExcessiveFileUploadSizeLimitCheck.yaml", new ExcessiveFileUploadSizeLimitCheck());
+  }
+
```

Comment:
Can you add (a) test case(s), where there a value with a suffix is raised in the file?